dev-infra/utils/git/index.ts

```diff
@@ -156,9 +161,10 @@ export class GitClient<Authenticated extends boolean> {
     }
 
     // To improve the debugging experience in case something fails, we print all executed Git
-    // commands to better understand the git actions occuring. Depending on the command being
-    // executed, this debugging information should be logged at different logging levels.
-    const printFn = (!this.verboseLogging || options.stdio === 'ignore') ? debug : info;
+    // commands at the DEBUG level to better understand the git actions occuring. Verbose logging,
+    // always logging at the INFO level, can be enabled either by setting the verboseLogging
+    // property on the GitClient instance or the options object provided to the method.
```

Comment:
```suggestion
    // property on the GitClient class or the options object provided to the method.
```

("Instance" makes is sound as if it is an instance property.)